src/directives/formly-form.js

```diff
@@ -16,7 +16,7 @@ module.exports = ngModule => {
       restrict: 'E',
       template: function(el, attrs) {
         /* jshint -W033 */ // this because jshint is broken I guess...
-        var rootEl = attrs.rootEl || 'ng-form';
+        var rootEl = attrs.rootEl || 'form';
```

Comment:
I would rather not overload the `rootEl` attribute to do this. I think the current implementation allows you to deal with the issue just fine. If you want to use an `ng-form` but you don't want it to be the root element, then you can do:

```html
<div ng-form>
  <formly-form root-el="div">
  </formly-form>
</div>
```